python/tests/test_jwt_cookie_csrf.py

```
@@ -0,0 +1,198 @@
+    async def header_write(request: dict):
+        return {"ok": True}
+
+    @api.post(
+        "/mixed-write",
+        auth=[
+            JWTAuthentication(secret=SECRET, cookie="access_token"),
+            JWTAuthentication(secret=SECRET),
+        ],
+        guards=[IsAuthenticated()],
+    )
+    async def mixed_write(request: dict):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `mixed_write`

(ANN202)